cmd/metal-api/main.go

```diff
@@ -493,14 +493,15 @@ func initIpam() {
 		if err != nil {
 			logger.Errorw("cannot connect to db in root command metal-api/internal/main.initIpam()", "error", err)
 			time.Sleep(3 * time.Second)
-			goto tryAgain
```

Comment:
now metal-api will not start anymore if ipamdb is not there ?

Reply:
is probably ok

Reply:
Yeah, but there's no functional change here at all